setup-nextcloud-app/action.yml

```diff
@@ -4,11 +4,15 @@ inputs:
   app:
     description: 'App name to test'
     required: true
-    default: null
+    default: 'null'
```

Comment:
Why all the defaults in quotes?

Reply:
vscode complained about them not being strings, turned out GitHub says they need to be strings https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault

Reply:
In that case the check will need to be for `!== 'null'` because it will always be something.

Reply:
Or we just drop the default

Reply:
That's also fine by me, though for the boolean defaults that doesn't help.

Reply:
But didn't they get converted to string since the beginning anyway?
If the parser says it needs to be a string. So probably it was never working the way you expected.

Reply:
```
function getInput(name: string, options?: core.InputOptions): string
Gets the value of an input. The value is also trimmed.

@param name — name of the input to get

@param options — optional. See InputOptions.

@returns — string
```

Reply:
I guess we can parse the expected type at the input, if js can do that

Reply:
TIL, yeah that was never working (but I would like it to after this PR)

Reply:
Yea I removed the default for the app name as that is required by the yaml anyway.

for the other two I set the default to false but they are not required else we wouldn't need a default value.

I'm not sure why there is a required option in js if it's already in the yaml, I will try to figure that out.

Reply:
> I'm not sure why there is a required option in js if it's already in the yaml, I will try to figure that out.

Ah I guess because you may call them directly from another action